src/main/java/io/fabric8/maven/docker/config/RunImageConfiguration.java

```diff
@@ -263,18 +264,14 @@ public VolumeConfiguration getVolumeConfiguration() {
         return volumes;
     }
 
-    public List<String> getLinks() {
-        return links;
+    public Iterable<String> getLinks() {
```

Comment:
Can we leave `List<String>` as return type ? All the other methods in RunImageConfiguration return a `List` for collections, and also it is not the most generic type which can be used here, I would like to keep it uniform.